backend/src/modules/notifications/listeners/others.listener.ts

```diff
@@ -1,21 +1,52 @@
 import { Injectable } from '@nestjs/common';
 import { OnEvent } from '@nestjs/event-emitter';
+import { VolunteerStatus } from 'src/modules/volunteer/enums/volunteer-status.enum';
+import { VolunteerFacade } from 'src/modules/volunteer/services/volunteer.facade';
+import { GetOneAnnouncementUseCase } from 'src/usecases/announcement/get-one-announcement.usecase';
+import { GetManyVolunteersUseCase } from 'src/usecases/volunteer/get-many-volunteers.usecase';
 import { EVENTS } from '../constants/events.constants';
 import SendAnnouncementEvent from '../events/others/send-announcement.event';
 
 @Injectable()
 export class OthersListener {
```

Comment:
Also Others is rather ambigous, what does this mean. As far as i can see it's just sendAnouncement Notification.